src/extension/features/accounts/last-reconciled-date/index.js

```diff
@@ -19,31 +19,68 @@ export class LastReconciledDate extends Feature {
     // Get the current account id and calculate the last reconciled date
     let { selectedAccountId } = controllerLookup('accounts');
     let latestDate = this._calculateLastReconciledDate(selectedAccountId);
-    let textToShow = 'NA';
-    if (latestDate) {
-      textToShow = ynab.YNABSharedLib.dateFormatter.formatDateExpanded(latestDate.utc());
-    }
-    // Retrieve or create the reconcile date container
-    let dateContainer = $(`#${TK_LAST_RECONCILED_ID}`);
-    if (!dateContainer || dateContainer.length === 0) {
-      $(YNAB_ACCOUNTS_HEADER_RIGHT).append(
-        `<div class="tk-accounts-header-last-reconciled">
-        <span id="${TK_LAST_RECONCILED_ID}">${textToShow}</span>
-        <div class="tk-accounts-header-last-reconciled-label">Last Reconciled Date</div>
-      </div>`
-      );
+
+    // Handle days since reconciled
+    if (this.settings.enabled.includes('days-since')) {
+      let daysSinceTextToShow = 'NA days';
+
+      // Get the current account id and calculate the last reconciled date
+      if (latestDate) {
+        // Calculate number of days since last reconciled
+        const oneDay = 24 * 60 * 60 * 1000; // hours*minutes*seconds*milliseconds
```

Comment:
Seems like we can use something like https://momentjs.com/docs/#/displaying/difference/ to calculate the difference for us

Reply:
done